packages/commonwealth/client/scripts/views/pages/landing/index.tsx

```diff
@@ -1,82 +0,0 @@
-import React from 'react';
-
-import 'pages/landing/index.scss';
-
-import CommunityInfo from '../../../models/ChainInfo';
-
-import useForceRerender from 'hooks/useForceRerender';
-import app, { LoginState } from 'state';
-import { Footer } from '../../Footer';
```

Comment:
do we use footer anywhere else than landing? probably something to delete as well, please check

Reply:
We use Footer in the sublayout